### ISSUE TITLE
Play opus files

### ISSUE THREAD

**Issue by XolbyNax** [KRTirtho/spotube#1324]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Pain

I have 512 opus files, and none can be played when placed in \Downloads\Spotube.

### Suggested solution

Since #691 has been rejected, at least let us play the files we have.

### Useful resources

https://wiki.xiph.org/OpusFAQ
https://opus-codec.org/
https://claude.ai/
https://chat.openai.com/
https://gemini.google.com/

### Additional information

Who hurt you? Tell me and I'll deal with them.
Please, stop discriminating against opus. 
I'll even start world war 3 if that's  what you want to add support for opus files.
I don't want to download music as opus, although that would be nice since I could uninstall newpipe, I just want to play what I already have. 

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by fxzxmicah** [comment]
I don't think spotube is a local file player.

**Comment by XolbyNax** [comment]
> I don't think spotube is a local file player.

Then you're in for a surprize, because as a matter of fact, spotube is a local file player, even if not marketed as such. 
Go to settings, and find **Download location** (\Downloads\Spotube by default). Grab any .mp3 and .m4a files you want and paste them in that directory, or change the directory to your local library, then, in Spotube go to library, and Local tracks (pc) or Tracks (Android) (someone should report this difference in naming), and click refresh, and you can see and play those files. 
![image](https://github.com/KRTirtho/spotube/assets/130172696/18f748e3-fd04-45e9-896a-e7f6d70e6d19)
![image](https://github.com/KRTirtho/spotube/assets/130172696/5e1b7280-74c6-4076-be76-96f6e4141ba6)

**Comment by fxzxmicah** [comment]
Interesting, I didn't know that.

**Comment by fabiscafe** [comment]
Is opus by now even supported on any platform? Last time i checked Android, Windows and <everything apple> where a huge problem.

**Comment by XolbyNax** [comment]
> Is opus by now even supported on any platform? Last time i checked Android, Windows and where a huge problem.

TL;DR:
Yes, it is supported by many apps, small open source projects, and corporations, on all major platforms.

apple music (macos/ios/android/windows/linux)
Microsoft's media player (windows)
VLC (windows/android/linux)
Aimp (windows/android/at least Arch linux)
Mx player (only android, I guess, not gonna look any further)
retro music player (android)
musicolet music player (android)
Tauon Music Box (linux)
Nuclear music player (windows/linux)

This is just a small sample. Flathub on linux has many players, and I could spend hours tryig them all to prove my point, and the play store is more of the same. Sadly, it's already past midnight, and I'd like to sleep sometime soon.

**Comment by KRTirtho** [comment]
Implemented 7cde803bee309f2107931869a9005b76224c4fce
